spec/formatting.md

```diff
@@ -24,6 +34,14 @@ Formatting of a _message_ is defined by the following operations:
   The resolution of _text_ is rather straightforward,
   and is detailed under _literal resolution_.
 
+Formatter implementations are not required to expose
```

Comment:
Too specific?

```suggestion
Implementations are not required to expose
```

Reply:
Sure; this paragraph was not actually modified, it just gets highlighted that way due to moving the note lower down.